crates/ty/src/logging.rs

```diff
@@ -24,15 +24,32 @@ pub(crate) struct Verbosity {
         help = "Use verbose output (or `-vv` and `-vvv` for more verbose output)",
         action = clap::ArgAction::Count,
         global = true,
+        overrides_with = "quiet",
     )]
     verbose: u8,
+
+    #[arg(
+        long,
+        help = "Use quiet output",
+        action = clap::ArgAction::Count,
+        global = true,
+        overrides_with = "verbose",
+    )]
+    quiet: u8,
```

Comment:
I'd prefer to keep this as a `bool` flag. It seems more confusing to users if they can use `-qq` but it doesn't change the behavior in anyway

Reply:
>  I think we'll want a --qq silent mode

What does this do? Does it silence everything? 

I don't feel too strongly. I just think both approaches are forward compatible and I'm not entirely sure yet if we indeed want `-qq` long term